test/pagination/paginate.spec.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable indent */
+
+const request = require('supertest');
+const { defaultPagParams } = require('../helpers/default_pag_params');
+const { createServer } = require('../server');
+const { customPagParams } = require('../helpers/custom_pag_params');
+const factory = require('../factory/fake_model');
+const { INVALID_PAGE_NUMBER, INVALID_LIMIT_NUMBER } = require('../../lib/pagination/errors');
+
+const testUrls = {
+  index: '/index',
+  indexWithParams: '/index_with_params',
+  indexPageException: '/index_page_exception',
+  indexLimitException: '/index_limit_exception'
+};
+const makeRequest = (url, content, method = 'get') => request(createServer(content))[method](url);
+
+describe.each`
+  description                             | testUrl                     | minSlice | maxSlice
+  ${'default pagination without options'} | ${testUrls.index}           | ${0}     | ${25}
+  ${'custom pagination with options'}     | ${testUrls.indexWithParams} | ${10}    | ${15}
+`('Request $description passed', ({ description, testUrl, minSlice, maxSlice }) => {
+  let fakeModels = undefined;
```

Comment:
Linter forces to do this ? Because is the same as `let fakeModels;`

Reply:
yes